Test/Tasty/Golden.hs

```diff
@@ -72,7 +72,8 @@ import Control.Applicative
 import Control.DeepSeq
 import qualified Data.Set as Set
 
--- | Compare a given file contents against the golden file contents
+-- | Create an output file and then compare its contents against the golden
+-- file's contents.
```

Comment:
I don't think this explanation is correct. It seems to imply that this function will create the file—it won't. It only does the "compare" part.